src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -612,7 +612,7 @@ class NexusPublishPluginTests {
             }
             nexusPublishing {
                 repositories {
-                    sonatype()
+                    sonatype
```

Comment:
`sonatype()` looks strange from Groovy point of view

for instance, you can't "create configuration" by calling

```
configurations {
  myConfig()
}
```

Reply:
Ah. The reasoning with `mavenCentral()` makes sense indeed.
However, I have no idea how to support `sonatype()` in Groovy DSL though.

Reply:
> Ah. The reasoning with `mavenCentral()` makes sense indeed.
> However, I have no idea how to support `sonatype()` in Groovy DSL though.

In that case, it's a non backward compatible change and we need to clearly state it in the release notes (also in README?) and probably bump the major version :-/. Because of that, I would prefer to first release a version containing some other fixes and improvements, so this PR is going to keep unmerged until then.